pkgs/os-specific/linux/vmware/default.nix

```
@@ -9,15 +9,13 @@
 
 stdenv.mkDerivation {
   pname = "vmware-modules";
-  version = "workstation-17.6.1-unstable-2024-10-12-${kernel.version}";
+  version = "workstation-17.6.3-${kernel.version}";
```

Nit: don't need to repeat the version

```suggestion
  version = "${vmware-workstation.version}-${kernel.version}";
```

---

The repo from which we get the vmware modules is third party (for mainline kernel compatibility patches) so it might not always have the latest version corresponding to the vmware workstation release (even though it *might* still work even if it's mismatched a little)

---

I see, so if someone really wants to use an older version, they'd need to pin the kernel too (or maintain their own patches), I was not aware of this tight integration, thanks for the info